mobile/mobile-core/devices-service.ts

```diff
@@ -672,6 +686,17 @@ export class DevicesService extends EventEmitter implements Mobile.IDevicesServi
 		shouldReturnImmediateResult = shouldReturnImmediateResult || false;
 		return { shouldReturnImmediateResult: shouldReturnImmediateResult, platform: platform };
 	}
+
+	private getEmulatorError(error: Error, platform: string): string {
+		let emulatorName = "Emulator";
+		if (this.$mobileHelper.isiOSPlatform(platform)) {
+			emulatorName = "Simulator";
+		}
+
+		return `Cannot find connected devices.${EOL}` +
+			`${emulatorName} start failed with: ${error.message}${EOL}` +
+			`To list currently connected devices and verify that the specified identifier exists, run '${this.$staticConfig.CLIENT_NAME.toLowerCase()} device'.`;
```

Comment:
Suggestion: `tns device` only provides the user with the information about the running devices, but the `--device` also accepts emulator image names. By that logic we can either tell the user to run `tns device android --available-devices` or make the `tns device` command to show the available emulator images.